apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/RequestResponseLoggingFilter.java

```diff
@@ -60,29 +60,37 @@ private static void handleRequest(ServletRequest request) {
     // Record the request type.
     MDC.put(computeMdcKey("request_type"), request.getClass().getName());
 
+    // Set the default Operation (will hopefully be customized further in specific handler methods).
+    Operation operation = new Operation(Operation.Endpoint.OTHER);
+
     if (request instanceof HttpServletRequest) {
       HttpServletRequest servletRequest = (HttpServletRequest) request;
 
       // Record the basic request components.
-      MDC.put(computeMdcKey("request.http_method"), servletRequest.getMethod());
-      MDC.put(computeMdcKey("request.url"), servletRequest.getRequestURL().toString());
-      MDC.put(computeMdcKey("request.uri"), servletRequest.getRequestURI());
-      MDC.put(computeMdcKey("request.query_string"), servletRequest.getQueryString());
+      operation = new Operation(Operation.Endpoint.matchByHttpUri(servletRequest));
+      MDC.put(computeMdcRequestKey("http_method"), servletRequest.getMethod());
+      MDC.put(computeMdcRequestKey("url"), servletRequest.getRequestURL().toString());
+      MDC.put(computeMdcRequestKey("uri"), servletRequest.getRequestURI());
+      MDC.put(computeMdcRequestKey("query_string"), servletRequest.getQueryString());
       MDC.put(
-          computeMdcKey("request.clientSSL.DN"),
+          computeMdcRequestKey("clientSSL.DN"),
           getClientSslPrincipalDistinguishedName(servletRequest));
 
       // Record the request headers.
       Enumeration<String> headerNames = servletRequest.getHeaderNames();
       while (headerNames.hasMoreElements()) {
         String headerName = headerNames.nextElement();
         List<String> headerValues = Collections.list(servletRequest.getHeaders(headerName));
-        if (headerValues.isEmpty()) MDC.put(computeMdcKey("request.header." + headerName), "");
+        if (headerValues.isEmpty())
+          MDC.put(computeMdcRequestKey("request.header." + headerName), "");
```

Comment:
Can't `request` in `request.header.` be omitted from the computeMdcRequestKey function call?

Reply:
Yup -- thanks! Fixed.